mindsdb/api/executor/command_executor.py

```diff
@@ -1048,7 +1048,7 @@ def _create_integration(self, name: str, engine: str, connection_args: dict):
             )
             handler_meta = handlers_meta[engine]
             if handler_meta.get("import", {}).get("success") is not True:
-                raise ExecutorException(f"Handler '{engine}' can not be used")
+                raise ExecutorException(f"The '{engine}' handler isn't installed.\n" + get_handler_install_message(engine))
```

Comment:
Maybe it makes sense to add `get_handler_install_message(engine)` only if it is not a cloud env.